ansible_rulebook/util.py

```diff
@@ -380,3 +380,33 @@ def validate_url(url: str, url_type: str) -> bool:
     if url_type == "controller":
         return res.scheme in ["http", "https"] and bool(res.netloc)
     return res.scheme in ["ws", "wss"] and bool(res.netloc)
+
+
+KEYS_TO_FILTER = [
```

Comment:
Can we blindly mask any key containing the word token or password?

Reply:
We could. I started out that way, but changed it to be more specific to not hinder a user if it used a custom variable. I can update to use that flow if we'd like too.
This is what Alex and I had aligned on currently, but happy to update